source/jobs/prerequisites/oracle.txt

```diff
@@ -86,38 +93,43 @@ Steps
                   GRANT SELECT ON V$ARCHIVED_LOG TO <user>;
                   GRANT SELECT ON V$ARCHIVE_DEST_STATUS TO <user>;
                   GRANT SELECT ON V$TRANSACTION TO <user>;
+                  GRANT SELECT ON V_$MYSTAT TO <user>;
+                  GRANT SELECT ON V_$STATNAME TO <user>; 
 
-         .. step:: Turn on ``LogMiner`` at the database level
-            
-            To run continous jobs against Oracle, you must enable 
-            ``LogMiner`` at the database level. The following code-block
-            is an example of automatically-generated code, which you 
-            can run manually by substituting your database name:
-
-            .. code-block:: sql
-                :copyable: true
-
-                ALTER SYSTEM SET db_recovery_file_dest_size = 10G;
-                ALTER SYSTEM SET db_recovery_file_dest = '/opt/oracle/oradata/recovery_area' SCOPE=spfile;
-                SHUTDOWN IMMEDIATE;
-                STARTUP MOUNT
-                ALTER DATABASE ARCHIVELOG;
-                ALTER DATABASE OPEN;
+         .. step:: Turn on archive logging
             
-            .. tip::
+            a. To check if archive logging is already enabled, run the following query:
+
+               .. code-block:: sql
+                  :copyable: true
 
-                To check if ``LogMiner`` is already enabled, run the 
-                following query:
+                  SELECT LOG_MODE FROM V$DATABASE;
 
-                .. code-block:: sql
-                   :copyable: true
+               This outputs ``ARCHIVELOG`` if logging is enabled, or
+               ``NOARCHIVELOG`` if it isn't.
+            
+            #. Enable archive logging if it isn't already enabled.
+            
+               The following is an example of the code Relational Migrator
```

Comment:
[question] How is the code generated? Where do I find it?

```suggestion
               Relational Migrator can automatically generate code to enable logging. 
```

Reply:
@marian-derias when does it generate this? I set up a test migration project on RM, but I'm guessing the Oracle test DB is already correctly configured, so I didn't see any generated code anywhere.

Reply:
Yes it doesn't get generated if the DB seems to be configured correctly, exactly. You can get that generated code if we detect that the database isn't correctly configured before you start the migration OR if the migration fails and it seems to be because the database isn't configured correctly. You'll get some button that says "Generate script" that lets you download a script with SQL or RDS commands. 

These are just recommendations of course and may not be exhaustive enough to correctly configure the database without DBA assistance.